examples/start_stop_server/start_stop_server.cpp

```diff
@@ -0,0 +1,36 @@
+//
+// Example how to use C API of mavsdk_server.
+//
+
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/mavsdk_server/mavsdk_server_api.h>
+#include <iostream>
+#include <csignal>
+
+using namespace mavsdk;
+
+static struct MavsdkServer* mavsdk_server;
+
+void signal_handler(int sig)
+{
+    std::cout << "Received SIGUSR1 signal" << std::endl;
+
+    mavsdk_server_stop(mavsdk_server);
+}
+
+int main(int argc, char* argv[])
+{
+    if (argc != 2) {
+        std::cout << "Connection string required as argument" << std::endl;
+        return 1;
+    }
+
+    signal(SIGUSR1, signal_handler);
```

Comment:
How is `SIGUSR1` triggered? I thought it would have to come programmatically, but in this example nothing sends it 🤔

Reply:
Actually, I should change it to the signal you get with Ctrl+C and catch that nicely.